commit 2419ec45d2e3513f65243a1113c300a3c554f710
Author: Stefan Kalkowski <stefan.kalkowski@genode-labs.com>
Date:   Wed Jul 12 14:23:23 2023 +0200

    Disable split transaction fix by default
    
    It doesn't work like this on Genode, therefore disable it for now.

diff --git a/drivers/usb/host/dwc_otg/dwc_otg_driver.c b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
index cb576e553..28d197718 100644
--- a/drivers/usb/host/dwc_otg/dwc_otg_driver.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
@@ -243,7 +243,7 @@ static struct dwc_otg_driver_module_params dwc_otg_module_params = {
 //Global variable to switch the fiq fix on or off
 bool fiq_enable = 1;
 // Global variable to enable the split transaction fix
-bool fiq_fsm_enable = true;
+bool fiq_fsm_enable = false;
 //Bulk split-transaction NAK holdoff in microframes
 uint16_t nak_holdoff = 8;
 
